src/query/functions/src/scalars/other.rs

```diff
@@ -384,6 +388,34 @@ fn register_grouping(registry: &mut FunctionRegistry) {
     })
 }
 
+fn register_num_to_char(registry: &mut FunctionRegistry) {
+    registry.register_passthrough_nullable_2_arg::<Int32Type, StringType, StringType, _, _>(
```

Comment:
let's use `Int64Type` cause every int type can auto-cast into i64.

Reply:
to_char needs to accept all numeric types.

which covers  `decimal type`, `float32`, `float64`, `ints`